insecure/corruptible/network_common_test.go

```diff
@@ -0,0 +1,148 @@
+package corruptible
```

Comment:
Same comment as the suggestions above use unittest.NetworkCodec() anywhere you are using the cbor.Codec() func in the tests.

Reply:
replaced all occurrences https://github.com/onflow/flow-go/pull/2811/commits/0e907c3ee2f107ba12caadf37240ae64507b9c89